src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Server.Json;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Operations
+{
+    internal class GetReplicationConflictsOperation : IMaintenanceOperation<BlittableArrayResult>
```

Comment:
let's avoid `BlittableArrayResult` and use something like
```
public class ConflictResults
{
 public ConflictResult[] Results;
 public long TotalResults;
}

public class ConflictResult
{
   public string Id;
   public string LastModified;
   ...
}
```

Reply:
I think we already have this here `GetConflictsResult`

Reply:
done